scripts/unbonding_batches_visualisation/src/app.ts

```diff
@@ -0,0 +1,192 @@
+import { SigningCosmWasmClient } from "@cosmjs/cosmwasm-stargate";
+import { DirectSecp256k1HdWallet } from "@cosmjs/proto-signing";
+import { GasPrice } from "@cosmjs/stargate";
+import {
+  Client as DropCoreClient,
+  UnbondBatch,
+} from "../../../integration_tests/src/generated/contractLib/dropCore";
+
+/* There're 2 possible modes for script
+ * RECENT will retrieve all recent batches since it'll meet one 'withdrawn'
+ * FULL will retrieve all batches from current batch to 0-nth
+ */
+enum Mode {
+  RECENT,
+  FULL,
+}
+
+const MODE: string = process.env.MODE;
+const CORE_CONTRACT: string = process.env.CORE_CONTRACT;
+const NODE_ADDRESS: string = process.env.NODE_ADDRESS;
+const WALLET_MNEMONIC: string = process.env.WALLET_MNEMONIC;
+
+/* addLeadingZeros used there to add leading zeros to date
+ * We need date formatting for pretty output
+ */
+function addLeadingZeros(num: number, targetLength: number): string {
+  let numStr: string = num.toString();
+  while (numStr.length < targetLength) {
+    numStr = "0" + numStr;
+  }
+
+  return numStr;
+}
+
+/*
+ * batch_id - number of batch in contract's order
+ * status - batch.status
+ * expected_amount - batch.expected_amount
+ * creation_time - batch.creation_time
+ * expected_finalization_time - (batch.creation_time + core.config.unbonding_period / 7)
+ * unstaked_amount - batch.unbonded_amount
+ */
+type BatchInfo = {
+  batch_id: number;
+  status: string;
+  expected_amount: number;
+  creation_time: string;
+  expected_finalization_time: string;
+  unstaked_amount: number;
+};
+
+/* Function print_n serves for getting information about 'n' first batches
+ * current_unbond_batch - latest unbonding batch gotten from query
+ * n - first n batches starting from current_unbond_batch
+ * drop_client - drop client generated code from binary, used for queries
+ */
+async function print_n(
+  current_unbond_batch: number,
+  n: number,
+  drop_client: DropCoreClient
+): Promise<Array<BatchInfo>> {
+  if (current_unbond_batch - n < 0) {
+    return [];
+  }
+  const dropCoreConfig = await drop_client.queryConfig();
+  let arr = [];
```

Comment:
please use more meaningful name. Also I suspect that this array should have `BatchInfo` type